frontend/tailwind.config.js

```diff
@@ -130,22 +108,15 @@ module.exports = {
       ],
     },
     fontSize: {
-      '13': '13px',
-      '14': '14px',
-      '15': '15px',
-      '16': '16px',
-      '17': '17px',
-      '18': '18px',
-      '20': '20px',
-      '22': '22px',
-      '24': '24px',
-      '30': '30px',
-      '35': '35px',
-      '40': '40px',
-      '50': '50px',
-      '62': '62px',
-      '66': '66px',
-      '70': '70px',
+      'base': '10px', 
+      'lg-base': '12px', 
+      'body': '16px',
+      'small': '14px',
+      'large': '18px',
+      'large-1': '4.7rem',
```

Comment:
I used this so we can set font-sizes without linking them to an h*.
The idea is that we set h1 to default to text-large-1. But we can use text-large-1 on any other element that needs the same size without it causing horrors like `<h2 class="h1">` :-)

Reply:
I think it's a good approach.

Reply:
I kept the body, small and large in px because I don't think they should scale, only the headings scale as the screen grows. I'll check this with Ben.

Reply:
For accessibility, if we'd have to add a button to scale the font sizes we'll need to scale every single font (e.g. what we did on Tenement Museum). But yes, we can review this with Ben.

Reply:
Ben's comments:
> We usually think about body text as fixed because the size recommendations are pretty consistent (14-18px), as opposed to headings where you can basically use the scale that you want. That being said, if there is a way to ensure that the body size doesn’t go bellow 14px and above 18px (while still allowing people with disabilities to scale up via browser settings) then making it relative sounds fine.

Reply:
If we have the 2 breakpoints with the html font size at 10px and 12px, then we could have:

- body: 1.5rem => 15px / 18px
- small: 1.3rem => 13px / 15.6px
- large: 1.7rem => 17px / 20.4px

We could also increase the html font size to give more fine grain control. Eg if we set it to 20px on mobile and 23px on desktop we could set the body to .75rem => 15px / 17.25px.

What do you think @Ben?

Would this work?

We could also play with the html font size and rem's to allow a bit 


set the body size to 1.5rem which would compute to 15px and 18px respectively. It does mean the small would be 12px and 17px, and the large

Reply:
@benbalderas ...

Reply:
The first approach looks better to me, having the html font size at 10px and 12px. The 2nd one could cause some issues with calculations.

Reply:
@adamf321 I also think the first approach can work, but we actually have 3 body font sizes, sorry for the missing info before:

Small: 14px - goes inside confined spaces like cards or other blocks
Regular: 16px - heroes, content blocks, etc
Large: 18px - exclusively for articles / blog posts

Then, if we make it fluid these sizes could go larger or smaller than we need them to be no?
This isn't a problem with headings.

@nelsonamaya82

Reply:
Ok on the 3 sizes. Let's just call them small, regular and large to avoid confusion (I used the word body for "regular", but really this was confusing).

It all depends how we configure them, but they aren't really fluid for now - there are just 2 options depending on screen size (though we could add more). My original proposal would mean they can vary as follows:
regular: 1.5rem => 15px / 18px
small: 1.3rem => 13px / 15.6px
large: 1.7rem => 17px / 20.4px

... but we can adjust this as you see fit by changing the html font size, breakpoints and rem of each size.

Reply:
So the the sizes will still be "fixed", but in relative units (preparing to be fluid in a near future)?

Reply:
Yes, exactly.